### PR TITLE
Inline script formatting

### PR DESCRIPTION
**Pull request by PolinaShneider**
Transferred PR from axibase/axibase-charts-vscode#363

### CODE REVIEW

File: src/regExpressions.ts [hunk 1]
```diff
@@ -100,6 +100,9 @@ export const ONE_LINE_COMMENT: RegExp = /\/\*([\s\S]*?)(?=\*\/)/;
 // number of spaces until first non-space character -  "   hello" // 3
 export const SPACES_AT_START: RegExp = /[^ ]/;
 
+// matches 1+ consequent spaces not surrounded by quotes
+export const UNQUOTED_CONSEQUENT_SPACES: RegExp = /(\s{2,})+(?=([^"]*"[^"]*")*[^"]*$)/;
```
Comment (VeselovAlex) [inline src/regExpressions.ts]: Try `((['"])(?:\\.|[^\2])*?\2)|(\s\s+)` — matches string literal (group 1) or whitespace sequence (group 3)

Comment (PolinaShneider) [reply src/regExpressions.ts]: Thank you, works like a charm.

File: src/test/indents.test.ts [hunk 2]
```diff
@@ -414,7 +414,7 @@ starttime = 2018
   });
 
   // TODO: will be affected when inline-scripts formatting PR is merged
-  test("Does not affect equals signs in setting value", () => {
+  test.skip("Does not affect equals signs in setting value", () => {
```
Comment (annstriganova) [inline src/test/indents.test.ts]: Since this case is covered in `scriptFormatting.test.ts`, remove it.

File: src/jsFormatter.ts [hunk 3]
```diff
@@ -1,29 +1,34 @@
...
 import { LanguageFormatter, LanguageFormattingOptions } from "./nestedCodeFormatter";
 
 export class JSFormatter implements LanguageFormatter {
     public language: string;
 
-    public format(unformattedCode: string, formattingOptions: LanguageFormattingOptions): string {
+    public format(unformattedCode: string, {
+        base, style, adjustMultilineComment = false,
+        semicolons = true, newline = "\n"
+    }: LanguageFormattingOptions): string {
```
Comment (annstriganova) [inline src/jsFormatter.ts]: Could it be moved to `const DEFAULT_FORMATTING_OPTIONS` to improve readability?

Comment (PolinaShneider) [reply src/jsFormatter.ts]: I caught your idea. Just thought that `DEFAULT_FORMATTING_OPTIONS` is a better name for formatting constants. 

Now working on extending `escodegen` default values for code formatting.

**Pull request merged**